### ISSUE TITLE
[EXP][upernet_swin_l_ce-1.0-dice-3.0] config 파일 업로드 및 실험 결과

### ISSUE THREAD

**Issue by seungriyou** [boostcampaitech3/level2-semantic-segmentation-level2-cv-10#53]
## 실험 주제 및 개요
`loss_decode`를 **cross entropy : dice loss = 1 : 3** 비율의 multiple loss로 설정한다.

## 추가 설명
`decode_head`와 `auxiliary_head`의 `loss_decode` 부분을 multiple loss(cross entropy : dice loss = 1 : 3)로 설정하였다.
```python
decode_head=dict(
    in_channels=[192, 384, 768, 1536], 
    num_classes=11,
    loss_decode=[dict(type='CrossEntropyLoss', loss_name='loss_ce', use_sigmoid=False, loss_weight=1.0),
                dict(type='DiceLoss', loss_name='loss_dice', use_sigmoid=False, loss_weight=3.0)]),
auxiliary_head=dict(
    in_channels=768, 
    num_classes=11,
    loss_decode=[dict(type='CrossEntropyLoss', loss_name='loss_ce', use_sigmoid=False, loss_weight=1.0*0.4),
                dict(type='DiceLoss', loss_name='loss_dice', use_sigmoid=False, loss_weight=3.0*0.4)]))
```

## 실험 결과 요약
- max epoch을 50으로 설정했으나, CV mIoU가 오르지 않아 best epoch은 32였다.
- epoch 30대의 모델을 제출하여서 LB mIoU가 낮게 나온 것 같기도 하다.
- 또한, 현재까지의 실험 결과에 따르면 cross entropy의 성능이 좋았기 때문에 **cross entropy : dice loss = 3 : 1로 ce의 비중을 높여봐도 좋을 것** 같다.

|  best epoch  |  32  |
|---|---|
|  CV mIoU  |  0.7080  | 
|  LB mIoU  |  0.6793  | 

```
2022-05-06 08:06:27,335 - mmseg - INFO - per class results:
2022-05-06 08:06:27,337 - mmseg - INFO -
+---------------+-------+-------+
|     Class     |  IoU  |  Acc  |
+---------------+-------+-------+
|   Background  | 96.62 | 98.93 |
| General trash | 47.23 | 57.52 |
|     Paper     |  79.8 | 88.19 |
|   Paper pack  | 65.07 | 78.38 |
|     Metal     | 66.11 | 71.42 |
|     Glass     | 66.74 | 73.02 |
|    Plastic    | 52.92 | 66.24 |
|   Styrofoam   | 72.36 | 78.58 |
|  Plastic bag  | 85.61 | 94.07 |
|    Battery    | 80.88 | 99.24 |
|    Clothing   | 65.47 | 74.42 |
+---------------+-------+-------+
2022-05-06 08:06:27,337 - mmseg - INFO - Summary:
2022-05-06 08:06:27,337 - mmseg - INFO -
+-------+------+------+
|  aAcc | mIoU | mAcc |
+-------+------+------+
| 94.57 | 70.8 | 80.0 |
+-------+------+------+
```
